README.md

```diff
@@ -373,7 +373,7 @@ To use your own credentials, you can setup the following environment variables:
 * [x] add support for gitbucket (cf [#142](https://github.com/guyzmo/git-repo/issues/142))
 * [ ] add support for managing SSH keys (cf [#22](https://github.com/guyzmo/git-repo/issues/22))
 * [ ] add support for issues (cf [#104](https://github.com/guyzmo/git-repo/issues/104))
-* [ ] add support for gerrit (cf [#19](https://github.com/guyzmo/git-repo/issues/19))
+* [x] add support for gerrit (cf [#19](https://github.com/guyzmo/git-repo/issues/19))
```

Comment:
IMO your first initial implementation of just `clone` isnt enough to mark this as 'done'.

I think this is done when git-repo can do the equivalent of `git review -d` & `git review` with a single patch in the queue. (patchset support isnt conceptually supported by git-repo)

Reply:
the minimal set of features to add a new service usually is: clone, fork, create, open, delete, add. In the context of gerrit, I'm not sure how all of them are relevant, but then not implementing those should be explicitly motivated.

Reply:
`fork` isnt a thing, at all.  People sometimes uses branches as an equivalent of forking.

As gerrit is a centrally managed fixed list of projects, `delete` and `create` would be super-user only operations. (very nice, but nobody would use them)

`open` would be a good addition to this first PR, IMO, as it requires that git-repo knows enough about the clone that it can fetch more metadata from the server.

Reply:
Okay. I will remove "done" mark as this PR introduces only basic functionality